tests/tests/operation_tests.cpp

```diff
@@ -6688,22 +6681,22 @@ BOOST_FIXTURE_TEST_SUITE(operation_tests, clean_database_fixture)
 
             const account_object& bob_acc = db->get_account("bob");
             const account_object& alice_acc = db->get_account("alice");
-            BOOST_REQUIRE(alice_acc.delegated_vesting_shares == ASSET_GESTS(1e7));
-            BOOST_REQUIRE(bob_acc.received_vesting_shares == ASSET_GESTS(1e7));
-            BOOST_REQUIRE(bob_acc.available_vesting_shares(true) ==
+            BOOST_CHECK(alice_acc.delegated_vesting_shares == ASSET_GESTS(1e7));
```

Comment:
It's better to use BOOST_CHECK_EQUAL and similarity macros. It's print actual values when check fails.